packages/salesforcedx-vscode-soql/package.json

```diff
@@ -19,28 +19,45 @@
     "Programming Languages"
   ],
   "dependencies": {
+    "@salesforce/sobject-metadata": "0.1.4",
```

Comment:
I thought we agreed on having this be part of `@salesforce/core`, did something change ?

Reply:
Nothing changed, it’s just not there yet.